hw5/hw5.sql

```diff
@@ -0,0 +1,137 @@
+use baseball;
+
+CREATE OR REPLACE TABLE Features as
+WITH HOME_TEAM as
+
+(
+SELECT
+    g.game_id,
+    g.local_date,
+    TBC.team_id,
+    'Home Team'  as HOME_AWAY,
+
+	SUM(TPC.Strikeout) AS Strikeouts,
+	SUM(TPC.plateApperance) as PlateApperance,
+
+	SUM(TPC.Single) as Single,
+	SUM(TPC.`Double`) as Double_,
+	SUM(TPC.Triple) as Triple,
+
+	COALESCE(ROUND(SUM(TBC.toBase) /SUM(NULLIF(TBC.atBat,0)),2),0) as Slugging_percentage,
```

Comment:
Good use of Coalesce. This way you can avoid dealing with Null values in python. The benefit of this is that you are making sure your data is high quality before importing it into python, which will save you the hassel of data wrangling.